db/schema.rb

```diff
@@ -10,17 +10,31 @@
 #
 # It's strongly recommended that you check this file into your version control system.
 
-ActiveRecord::Schema.define(version: 20171024183246) do
+ActiveRecord::Schema.define(version: 20180209175817) do
 
   # These are extensions that must be enabled in order to support this database
   enable_extension "plpgsql"
 
   create_table "books", force: :cascade do |t|
     t.string "title"
+    t.bigint "rating_id"
+    t.index ["rating_id"], name: "index_books_on_rating_id"
+  end
+
+  create_table "ratings", force: :cascade do |t|
+    t.integer "rating"
+    t.bigint "book_id"
+    t.string "body"
+    t.bigint "user_id"
+    t.index ["book_id"], name: "index_ratings_on_book_id"
+    t.index ["user_id"], name: "index_ratings_on_user_id"
   end
```

Comment:
Looks like it took some round about migrations, but you got pretty close in the end. This table should be a reviews table, but you've figured out it should be a join table between book and user and also have a rating and a body on it.